common/util.c

```diff
@@ -140,8 +141,13 @@ expanduser(char *str)
 		continue;
 	if (t == p) {
 		/* ~ */
+#ifdef __GLIBC__
```

Comment:
This is hacky.  We had a request (#40 ) to use musl before.

Reply:
musl doesn't require code change here because it does support issetugid(). glibc on the other hand supports secure_getenv() but not issetugid()